src/components/Button/Button.story.jsx

```diff
@@ -14,6 +15,7 @@ stories.add(
       size: ['default', 'large'],
       color: ['default', 'primary', 'secondary'],
       disabled: [true, false],
+      icon: [null, ...Object.keys(iconNames)],
```

Comment:
after looking at the huge percy diff https://percy.io/Kids-First-DRC/KF-UIkit/builds/1732394/view/112838780/1280?mode=diff&browser=chrome&snapshot=112838780

I think it would be easier to test with only one icon variation and that should cover most cases so 
```js
icon: [null, Object.keys(iconNames)[0]]
```

Sorry for flip flopping on this